docker-compose.yml

```
@@ -0,0 +1,37 @@
+version: "3.7"
+services:
+  gesso:
+    build: .
```

Align this with https://github.com/forumone/gesso-wp/pull/240/files
```suggestion
    build: ./
    init: true
```